tpl/dash/dashboard.tpl.php

```diff
@@ -134,13 +134,35 @@
 	<?php if (!$__cloud->activated()) : ?>
 		<div class="litespeed-dashboard-unlock">
 			<div>
-				<h3 class="litespeed-dashboard-unlock-title"><strong class="litespeed-qc-text-gradient">Accelerate, Optimize, Protect</strong></h3>
-				<p class="litespeed-dashboard-unlock-desc">Speed up your WordPress site even further with <strong>QUIC.cloud Online Services and CDN</strong>.</p>
-				<p>Free monthly quota available.</p>
-				<p><a class="button button-primary" href="<?php echo Utility::build_url(Router::ACTION_CLOUD, Cloud::TYPE_ACTIVATE); ?>"><span class="dashicons dashicons-yes"></span>Enable QUIC.cloud services</a></p>
+				<h3 class="litespeed-dashboard-unlock-title">
+					<strong class="litespeed-qc-text-gradient">
+						<?php _e('Accelerate, Optimize, Protect', 'litespeed-cache'); ?>
+					</strong>
+				</h3>
+				<p class="litespeed-dashboard-unlock-desc">
+					<?php
+					sprintf(
+						__('Speed up your WordPress site even further with %sQUIC.cloud Online Services and CDN%s.', 'litespeed-cache'),
+						'<strong>',
+						'</strong>'
```

Comment:
It may sound strange but HTML tags are part of languages strings.

Reply:
done